pkgs/development/python-modules/python-hglib/default.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, python3Packages
+}:
+
+python3Packages.buildPythonPackage rec {
```

```suggestion
buildPythonPackage rec {
```

This is not allowed for pythonpackages because otherwise we would mix python versions.

---

Interesting! This is used in 32 python packages in total. Should we file an issue to fix those as well?